src/megachatapi.h

```diff
@@ -2230,15 +2230,21 @@ class MegaChatApi
      * of the app's cache, but may include new messages that wants to be imported into the app's
      * cache in one shot. In case the history has been truncated, this method applies truncation.
      *
+     * The associated request type with this request is MegaChatRequest::TYPE_IMPORT_MESSAGES
+     * Valid data in the MegaChatRequest object received on callbacks:
+     * - MegaChatRequest::getText - Returns the cache path
```

Comment:
We could add the count of imported messages to the request, so the app still can check. I'll do it.

Reply:
Fixed